README.md

```diff
@@ -29,10 +29,11 @@ NOTE: If you are a visual learner, then [this video tutorial](https://drive.goog
 - First, we need the URL for your github or gitlab repository.
   - This can be found from copying the URL from the root of your repository's main/master branch.
 - Go back to your slack channel from the previous step and execute the command: `/captainslog new project`.
-  - This should prompt you with a modal with 3 input fields.
+  - This should prompt you with a modal with 4 input fields.
```

Comment:
This quick start section is getting complicated. Might be time to break it off into its own document?

Reply:
I completely agree